From 45040e7d268329ebc40e6cb237c64a6637cfab5c Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 13 Mar 2017 20:22:10 -0700
Subject: [PATCH] config/libopts.m4: regenerate it from config/libopts.def

It was out of date compared to config/libopts.def, so regenerate it via
"autogen config/libopts.def" command.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 config/libopts.m4 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config/libopts.m4 b/config/libopts.m4
index c7ba4f3..51e6a39 100644
--- a/config/libopts.m4
+++ b/config/libopts.m4
@@ -2,7 +2,7 @@ dnl  -*- buffer-read-only: t -*- vi: set ro:
 dnl
 dnl DO NOT EDIT THIS FILE   (libopts.m4)
 dnl
-dnl It has been AutoGen-ed
+dnl It has been AutoGen-ed  March 13, 2017 at 08:21:21 PM by AutoGen 5.18
 dnl From the definitions    libopts.def
 dnl and the template file   conftest.tpl
 dnl
@@ -114,6 +114,7 @@ AC_DEFUN([INVOKE_LIBOPTS_MACROS_FIRST],[
   AC_PROG_SED
   [while :
   do
+      test -x "$POSIX_SHELL" && break
       POSIX_SHELL=`which bash`
       test -x "$POSIX_SHELL" && break
       POSIX_SHELL=`which dash`
-- 
2.10.2

